test/test_supervisor.rb

```diff
@@ -213,6 +214,52 @@ def test_main_process_signal_handlers
     $log.out.reset if $log && $log.out && $log.out.respond_to?(:reset)
   end
 
+  def test_cont_in_main_process_signal_handlers
+    omit "Windows cannot handle signals" if Fluent.windows?
+
+    opts = Fluent::Supervisor.default_options
+    sv = Fluent::Supervisor.new(opts)
+    sv.send(:install_main_process_signal_handlers)
+
+    begin
+      Process.kill :CONT, $$
+    rescue
+    end
+
+    sleep 1
+
+    assert_true File.exist?(@sigdump_path)
```

Comment:
Power assert style is preferred for better failure message:

```suggestion
    assert do
      File.exist?(@sigdump_path)
    end
```

`assert_true ...` shows just `File.exist?(@sigdump_path)` is not `true` on failure.
But `assert {...}` shows not only `File.exist?(@sigdump_path)` value but also `@sigdump_path` and so on.